lib/index.js

```diff
@@ -238,7 +260,7 @@ module.exports.renderSync = function(options) {
 
   if (importer) {
     options.importer = function(file, prev) {
-      return { objectLiteral: importer(file, prev) };
+      return importer(file, prev);
```

Comment:
This is true. We can remove this block from here (we would probably need to keep it in renderSync).